src/ring_theory/ideals.lean

```diff
@@ -350,10 +350,29 @@ end quotient
 section lattice
 variables {R : Type u} [comm_ring R]
 
+lemma mem_sup_left {S T : ideal R} : ∀ {x : R}, x ∈ S → x ∈ S ⊔ T :=
+show S ≤ S ⊔ T, from le_sup_left
```

Comment:
Can you avoid repeating this proof in both `ideals` and `linear_algebra.basic`?

Reply:
I could, since `ideal` is just defined to be `submodule`, but I don't think there's any harm in having both, it saves a bit of confusion, because people don't usually think about ideals as submodules, and it can get annoying having to remember which `ideal` lemmas are also true of `submodule`s when using a library set up like that.

Reply:
Sorry, what I mean is can you implement the proof in terms of the existing proof in `submodule`, like some of the others in this file.